### DISCUSSION TITLE
vue project, adding vitepress, dev server issue

### DISCUSSION THREAD

**Discussion by kyartscouncil** [vuejs/vitepress#1153] [Q&A]
I have an established vite/vue project, with vuerouter, pinia, etc.

I am attempting to add vitepress to this project but I'm having difficulty establishing a working dev environment.

The vite server option has disappeared and now the vitepress server option is running. But my vite project is not showing, and the vitepress server is giving a 404 error and:
`Component is missing template or render function. 
  at <Layout> 
  at <VitePressApp>`

Am I creating an abomination that is forever doomed or is there a simple fix?

**Comment by brc-dd** [comment]
Can you share your code?
